abayo-luc/TheySaid
# #166240371 Set Up Project  Repository

### What does this PR do?

Setup ReactNative project with test and continuous integration service 

### Description of Task Completed
- Initialize project with `create-react-native-app`
- Setup test env with jest
- Integrate circleci and code coverage
- Add eslint configuration

### How should this be manually tested
After cloning the repo, cd into it and run `npm install` then run `npm test` to see the test running
You can also click on the badge to see circleci and coveralls reports.

### What are the relevant pivotal tracker stories?
#166240371
#166257426
